### PR TITLE
Add English string for 'Donate to EFF'

### PR DESCRIPTION
**Pull request by Hainish**
Adding this initial English string so Transifex will pick it up and add it to the translations submodule for all other languages.

### CODE REVIEW

File: src/chrome/locale/en/https-everywhere.dtd [hunk 1]
```diff
@@ -14,6 +14,7 @@
 <!ENTITY https-everywhere.about.or "or">
 <!ENTITY https-everywhere.about.donate_eff "Donating to EFF">
 
+<!ENTITY https-everywhere.menu.donate_eff_imperitive "Donate to EFF">
```
Comment (jsha) [inline src/chrome/locale/en/https-everywhere.dtd]: *imperative

Comment (Hainish) [inline src/chrome/locale/en/https-everywhere.dtd]: whoops!

**Comment by jsha** [comment]
LGTM. Merge once tests pass.

**Comment by Hainish** [comment]
ping @jsha

**Pull request merged**